### PR TITLE
Stat effects

### LINKED ISSUE #498: More information on stats
Each worker stat has its own buff, instead of just contributing to worker level. This is mainly a reminder for me to add the individual buffs (use the spreadsheet, raven!)

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #498 

## Changes proposed:
- Adds effects of each individual stat
- Doesn't have the Archer In Training, Knight In Training, Library Student, Planter, or Pupil effects yet as those aren't on the spreadsheet

Merge please :D

### CODE REVIEW

**Review by ShadowProtocol — COMMENTED** [review]
As a general comment, there is a lot of duplication now: it may be worth considering talking about buildings and their workers on a single page each in future.

**Comment by ravenbuilder934** [comment]
> As a general comment, there is a lot of duplication now: it may be worth considering talking about buildings and their workers on a single page each in future.

The wiki was like this once a while back (I only know about it from reading the discord backlog). Ray wanted it the way it is now

File: source/buildings/courier.md [hunk 1]
```diff
@@ -21,9 +21,7 @@ layout: default
 
 The Courier runs back and forth from the [Warehouse](../../source/buildings/warehouse) to all the worker huts in your colony, delivering materials to workers and putting finished products in the Warehouse.
 
-**Note:** You *must* build the Warehouse to at least level 1, otherwise, the Courier won't be able to do their job.
-
-The level of the Courier's Hut will dictate how many items/blocks the Courier can carry to/from Huts-Warehouse-Huts, so if you want them to carry more, be sure to upgrade the Courier's Hut. Also, the higher the level of the hut, the faster the Courier will move between huts and the Warehouse.
+The level of the Courier's Hut will dictate how many items the Courier can carry to/from Huts - Warehouse - Huts, so if you want them to carry more materials/tools, upgrade their hut. Upgrading the hut will also increase the amount of requests they can keep track of at a time.
```
[the remark this answers is not in the document]
Comment (ravenbuilder934) [reply source/buildings/courier.md:24]: Doesn't this also include deliveries?

File: source/workers/rabbitherder.md [hunk 2]
```diff
@@ -37,4 +37,4 @@ The Rabbit Herder will work with vanilla Minecraft rabbits, but might work with
 | 4         | 8           |
 | 5         | 10          |
 
-<br>
+The greater a Rabbit Herder's Agility level, the higher their chance to successfully hit a rabbit (if they miss, they'll keep trying until the rabbit dies). The greater their Athletics level, the slower it'll take for rabbits to grow up.
```
[the remark this answers is not in the document]
Comment (ravenbuilder934) [reply source/workers/rabbitherder.md]: Facepalm. Thank you for catching this!

File: source/workers/courier.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ layout: default
 
 The Courier is a critial part of the bloodline of your colony. They will run from worker's hut to [Warehouse](../../source/buildings/warehouse) to worker's hut, visiting all your workers and delivering the tools and materials they need as well as taking their production to the Warehouse.
 
-The level of the Courier's Hut will dictate how much materials/tools the Courier can carry to/from Huts - Warehouse - Huts, so if you want them to carry more materials/tools, upgrade the Courier's Hut. Also, the higher the level of the hut, the faster the Courier will run.
+The level of the Courier's Hut will dictate how many items the Courier can carry to/from Huts - Warehouse - Huts, so if you want them to carry more materials/tools, upgrade their hut. Upgrading the hut will also increase the amount of requests they can keep track of at a time.
```
[the remark this answers is not in the document]
Comment (ravenbuilder934) [reply source/workers/courier.md:26]: Doesn't this also include deliveries?

File: source/buildings/courier.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShadowProtocol) [reply source/buildings/courier.md:24]: From my quick glance the AI uses the terms almost interchangeably. Maybe just go with deliver. Edited.

**Comment by ShadowProtocol** [comment]
> Ray wanted it the way it is now

I thought this might be the case. Perhaps it was expected that there would eventually be more content in each. But as it stands there's not enough unique details to justify it IMO. It just adds confusion for the user to find information. *"Was that fact I'm looking for on the building page, or the worker page?"* But maybe I'm just lazy and want less to maintain. Ah well.

**Comment by ravenbuilder934** [comment]
What are the skill effects for the planter?

**Comment by ShadowProtocol** [comment]
> What are the skill effects for the planter?

Normal crafter drill - Primary for improving recipes, Secondary for crafting them quicker. Agility and Dexterity respectively for the Planter.

**Comment by ravenbuilder934** [comment]
> > What are the skill effects for the planter?
> 
> Normal crafter drill - Primary for improving recipes, Secondary for crafting them quicker. Agility and Dexterity respectively for the Planter.

K, I figured it might have something to do with their crops which is why I asked. Is this ready to be merged now? Everything's accurate?

File: source/buildings/university.md [hunk 4]
```diff
@@ -53,6 +53,6 @@ Page 2 of the GUI will show you the options for each research tree.
     
 <br>
     
-Once you click on a research tree, you will see the options for each research branch. Each option will tell you the requirements and how long it takes to research that option. (Time is real-world time spent in-game.)
+Once you click on a research tree, you will see the options for each research branch. Each option will tell you the requirements and how long it takes to research that option. (Time is real-world time spent in-game. However, Researchers will sometimes use offline time to work on researches. Researchers' Mana skill affects the max amount of offline time they can use, and their Knowledge skill affects the amount of research hours they get from one hour of offline time.)
```
Comment (ShadowProtocol) [inline source/buildings/university.md:56]: I mean the thing's measured in seconds, so hours have very little to do with it. Generally you don't put a complete concept in brackets. Check out my original suggestion, you used a similar phrasing for the researcher now anyway.

**Review by ShadowProtocol — COMMENTED** [review]
Easy to miss something with this sort of thing, but yeah I'm pretty sure all the facts line up now.

**Pull request merged**